sharedgo/deletionPicker.go

```diff
@@ -0,0 +1,21 @@
+// Picks what file from 10 to 50 (files 0-9 kept in the system for persistence) to delete
+
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"strconv"
+	"time"
+)
+
+func pick_file_to_delete() string {
```

Comment:
Please add a comment explaining what this function does and any nuances to its operation (also useful for godoc!).

Godoc: https://godoc.org/golang.org/x/tools/cmd/godoc